packages/website/ts/@next/components/button.tsx

```diff
@@ -19,52 +19,55 @@ interface ButtonInterface {
     isInline?: boolean;
     href?: string;
     type?: string;
+    target?: string;
     to?: string;
     onClick?: () => any;
     theme?: ThemeInterface;
+    useAnchorTag?: boolean;
 }
 
 export const Button = (props: ButtonInterface) => {
-    const {
-        children,
-        href,
-        isWithArrow,
-        to,
-    } = props;
+    const { children, href, isWithArrow, to, useAnchorTag, target } = props;
     let linkElem;
 
-    if (href) { linkElem = 'a'; }
-    if (to) { linkElem = ReactRouterLink; }
+    if (href || useAnchorTag) {
```

Comment:
I know this is janky but I didn't want to make all items with `onClick` into `a`s, just hacking this together given the timeline

Reply:
I needed this to fix the alignment of one of the buttons